src/contracts/middleware/BLSSignatureChecker.sol

```diff
@@ -92,24 +107,34 @@ abstract contract BLSSignatureChecker {
         {
             // the quorumBitmaps of the nonSigners
             uint256[] memory nonSignerQuorumBitmaps = new uint256[](nonSignerStakesAndSignature.nonSignerPubkeys.length);
+            {
+                // the bitmap of the quorumNumbers
+                uint256 singingQuorumBitmap = BytesArrayBitmaps.bytesArrayToBitmap(quorumNumbers);
```

Comment:
not quite understanding this variable name

Reply:
haha fixed: https://github.com/Layr-Labs/eigenlayer-contracts/pull/74/commits/cbadd96f524f26254528ece2a95645c55c3edeb8